From fc55b203474c7b698f6788944502ac4e7306197c Mon Sep 17 00:00:00 2001
Message-Id: <patch-v4-1.6-fc55b203474-20220331T014349Z-avarab@gmail.com>
In-Reply-To: <cover-v4-0.6-00000000000-20220331T014349Z-avarab@gmail.com>
References: <cover-v3-0.7-00000000000-20220326T171200Z-avarab@gmail.com>
	<cover-v4-0.6-00000000000-20220331T014349Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 27 Dec 2021 04:12:43 +0100
Subject: [PATCH v4 1/6] run-command.h: remove always unused
 "clean_on_exit_handler_cbdata"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Remove a "struct child_process" member added in
ac2fbaa674c (run-command: add clean_on_exit_handler, 2016-10-16), but
which was never used.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 run-command.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/run-command.h b/run-command.h
index 07bed6c31b4..5bd0c933e80 100644
--- a/run-command.h
+++ b/run-command.h
@@ -142,7 +142,6 @@ struct child_process {
 	unsigned clean_on_exit:1;
 	unsigned wait_after_clean:1;
 	void (*clean_on_exit_handler)(struct child_process *process);
-	void *clean_on_exit_handler_cbdata;
 };
 
 #define CHILD_PROCESS_INIT { \
-- 
2.35.1.1561.ge8eddc63765

